include/drivers/fprotect.h

```diff
@@ -9,6 +9,19 @@
 
 #include <stdint.h>
 #include <string.h>
+#include <zephyr/types.h>
+
+/**
+ * @file
```

Comment:
Is empty file tag valid? Genuine question.

Reply:
I just copied it. It seems quite common.